tools/sdk/ld/eagle.app.v6.common.ld.h

```diff
@@ -229,6 +247,9 @@ SECTIONS
     /* Fundamental type info */
     *(.rodata._ZTIPKc .rodata._ZTIc .rodata._ZTIv .rodata._ZTSv .rodata._ZTSc .rodata._ZTSPKc .rodata._ZTSi .rodata._ZTIi)
 
+    /* std::make_shared */
+    *(.rodata._ZZNSt19_Sp_make_shared_tag5_S_tiEvE5__tag)
+
```

Comment:
Looking at shared ptr bits
https://github.com/gcc-mirror/gcc/blob/fee53a3194c0d8b747486b23980f1214cc1355b9/libstdc%2B%2B-v3/include/bits/shared_ptr_base.h#L566

feels weird leaving stdlib static here of a implementation-dependant obj. I'd rather remove this one